extensions/wikia/AssetsManager/config.php

```diff
@@ -3172,23 +3172,44 @@
 	]
 ];
 
-$config['article_video_scss'] = [
+$config['article_featured_video_scss'] = [
 	'type' => AssetsManager::TYPE_SCSS,
 	'skin' => [ 'oasis' ],
 	'assets' => [
 		'//extensions/wikia/ArticleVideo/styles/ooyala/html5-skin.scss',
-		'//extensions/wikia/ArticleVideo/styles/article-video.scss',
+		'//extensions/wikia/ArticleVideo/styles/article-featured-video.scss',
 	],
 ];
 
-$config['article_video_js'] = [
+$config['article_featured_video_js'] = [
 	'type' => AssetsManager::TYPE_JS,
 	'skin' => [ 'oasis' ],
 	'assets' => [
 		'//extensions/wikia/ArticleVideo/scripts/ooyala/core.js',
 		'//extensions/wikia/ArticleVideo/scripts/ooyala/main_html5.js',
 		'//extensions/wikia/ArticleVideo/scripts/ooyala/html5-skin.js',
 		'//extensions/wikia/ArticleVideo/scripts/ooyala-player.js',
-		'//extensions/wikia/ArticleVideo/scripts/article-video.js',
+		'//extensions/wikia/ArticleVideo/scripts/article-featured-video.js',
+	],
+];
+
+$config['article_related_video_scss'] = [
+	'type' => AssetsManager::TYPE_SCSS,
+	'skin' => [ 'oasis' ],
+	'assets' => [
+		'//extensions/wikia/ArticleVideo/styles/ooyala/html5-skin.scss',
+		'//extensions/wikia/ArticleVideo/styles/article-related-video.scss',
+	],
+];
+
+$config['article_related_video_js'] = [
+	'type' => AssetsManager::TYPE_JS,
+	'skin' => [ 'oasis' ],
+	'assets' => [
+		'//extensions/wikia/ArticleVideo/scripts/ooyala/core.js',
+		'//extensions/wikia/ArticleVideo/scripts/ooyala/main_html5.js',
+		'//extensions/wikia/ArticleVideo/scripts/ooyala/html5-skin.js',
+		'//extensions/wikia/ArticleVideo/scripts/ooyala-player.js',
```

Comment:
Can a featured and related video be on the same page? If yes, then we will send users a lot of duplicated code. Maybe we should reorganize it differently, especially that related video code is pretty small compared to the rest.